docs/README.md

```diff
@@ -166,6 +169,29 @@ documentation, please create a thread on the Swift forums under the
   <!-- NOTE: Outdated -->
 - [Lexicon.md](/docs/Lexicon.md):
   Canonical reference for terminology used throughout the project.
+  
+### ABI
+
+- [GenericSignature.md](/docs/ABI/GenericSignature.md):
+  Generic entities in Swift have a corresponding generic signature,
+  which describes a set of generic type parameters along with a set
+  of constraints on those parameters.
```

Comment:
I think it would be helpful if the description here (and the other ones below) briefly summarized the contents of the document. For example, say something like:

> Describes what generic signatures are and how they are used in the ABI, including the algorithm used for minimization and canonicalization.

Nit: Please start the description with a verb (describes/provides/outlines/summarizes/explains etc.), otherwise it reads a bit odd IMO.